test/tests/parser/param_substitution/unit_conversion.i

```diff
@@ -0,0 +1,37 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  solve = false
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Postprocessors]
+  [./km_to_m]
+    type = FunctionValuePostprocessor
+    function = ${units 1 km -> m}
```

Comment:
Can things like `${units 1 km -> m}` be used to replace any constant in an input file?

Reply:
Yes - the `units` brace expression just evaluates to a  number.  After resolving it, the input file above effectively looks like `function = 1000`.